client/app/_layout.js

```diff
@@ -13,20 +14,45 @@ import { SessionProvider } from '../context/auth';
 import { ThemeProvider } from '../context/theme';
 import FlashMessage from 'react-native-flash-message';
 import Footer from '../components/footer/Footer';
+import { queryTrpc, getToken } from '../trpc';
+import { httpBatchLink } from '@trpc/client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { api } from '~/constants/api';
 
 export default function HomeLayout() {
+  const queryClient = new QueryClient()
```

Comment:
we should move this to to its own TrpcQueryProvider context just for cleaner code